csharp/test/Drivers/Apache/Spark/DriverTests.cs

```diff
@@ -37,6 +37,38 @@ namespace Apache.Arrow.Adbc.Tests.Drivers.Apache.Spark
     [TestCaseOrderer("Apache.Arrow.Adbc.Tests.Xunit.TestOrderer", "Apache.Arrow.Adbc.Tests")]
     public class DriverTests : SparkTestBase
     {
+        /// <summary>
+        /// Supported Spark data types as a subset of <see cref="SparkConnection.ColumnTypeId"/>
+        /// </summary>
+        private enum SupportedSparkDataType : short
+        {
+            ARRAY_TYPE = 2003,
```

Comment:
Should we use ARRAY_TYPE = `ColumnTypeId.ARRAY_TYPE`? In case of value update, it would be required at only one location?

Reply:
Also, do we need `SupportedSparkDataType`. Should `ColumnTypeId` consist of only supported types instead?

Reply:
Leaving all types in ColumnTypeId.